README.md

```diff
@@ -42,6 +42,7 @@ You should file a [Pull Request](https://github.com/AlexsLemonade/OpenPBTA-analy
 We are using continuous integration software applied to the supplied test datasets to confirm that the analysis can be carried out successfully within the Docker container.
 
 ## How to Obtain OpenPBTA Data
+This dataset includes all the PBTA somatic mutational and gene expression results in combined tsv or matrix format. We will be releasing this dataset on both [CAVATICA](https://cavatica.sbgenomics.com) and AWS cloud platform moving forward.
```

Comment:
Some of this appears to be duplicated in the paragraph starting with line 47. Can you combine these two sets of sentences? 

In addition, can you put each sentence on its own line? That helps during review because a reviewer can comment on the specific sentence they want to comment on and it helps make the diffs more clear.

Reply:
thanks @jaclyn-taroni. i removed the duplicated words. and good suggestion on each sentence with its own line. will do that moving forward.